cfme/tests/containers/test_ssl_providers.py

```diff
@@ -130,16 +131,19 @@ def test_add_hawkular_provider_ssl(provider, appliance, test_item, soft_assert,
         """
     new_provider = copy(provider)
     new_provider.endpoints['default'].sec_protocol = test_item.default_sec_protocol
-    new_provider.endpoints['hawkular'].sec_protocol = test_item.hawkular_sec_protocol
+    new_provider.endpoints['metrics'].sec_protocol = test_item.metrics_sec_protocol
     try:
         new_provider.setup()
         view = appliance.browser.create_view(ContainerProvidersView)
         view.flash.assert_success_message(
             'Containers Providers "' + provider.name + '" was saved')
     except AssertionError:
-        soft_assert(False, provider.name + ' wasn\'t added successfully using ' +
-                    test_item.default_sec_protocol + ' security protocol and ' +
-                    test_item.hawkular_sec_protocol + ' hawkular security protocol')
+        soft_assert(False,
+                    "provider_name wasn\'t added successfully using {default_sec_protocol} "
```

Comment:
**REQUIRED**
If you use double quotes then no need to escape single quote.

Reply:
Could you be clearer? I don't use and single quote here

Reply:
Instead of `"provider_name wasn\'t added successfully using {default_sec_protocol} "`, just write `"provider_name wasn't added successfully using {default_sec_protocol} "`